backend/curriculum_tracking/helpers.py

```diff
@@ -182,3 +183,32 @@ def agile_card_reviews_outstanding(user):
 def pull_request_reviews_outstanding(user):
     # TODO: only implement this once the github webhook is healthier
     return []
+
+
+def add_users_to_team(team, email_addresses):
+    if team:
+        users = core_models.User.objects.filter(email__in=email_addresses)
+        if users:
+            team.user_set.add(*users)
+
+        users_added_to_team = [user.email for user in users]
+        return users_added_to_team
+
+    return []
+
+
+def remove_leading_and_trailing_whitespace(string):
+    if string and type(string) == str:
+        return string.strip()
+    return ""
+
+
+def get_email_addresses_from_str(email_addresses_str):
```

Comment:
Should this not be part of the form? Why is it all the way over here? It's not cohesive here. This function is only used on the form.

Also, the code can be a bit tighter. It's not necessary to separate tiny things out into new functions. It gets a bit spaghetti-like.

```
def get_email_addresses_from_str(email_addresses_str):
    email_addresses = re.split("[ ,\n]", email_addresses_str)
    email_addresses = [email.strip() for email in email_addresses]
    return [email for email in email_addresses if email]
```